AuctionHouse/Overview.md

```diff
@@ -0,0 +1,101 @@
+
+* **Bid + Sale in same transaction**: Alice lists an asset, Bob matches the offer and the sale goes through immediately.
+* **Offer + Sale in same transaction**: Bob makes an offer of 5 SOL for an unlisted asset owned by Alice, Alice likes the offer and lists the asset for 5 SOL thus completing the sale immediately.
+* **Offer, Bid, Sale in three separate instructions**: Alice lists an asset for 5 SOL, Bob makes an offer for this asset for 3 SOL, Alice likes the offer and agrees to the sale.
+    
+## Auctioning Fungible Assets
+An asset can be any SPL token and doesn't have to be an NFT ("Non-Fungible Token"). 
+
+The only constraint is that the Mint Account of the SPL Token need to have a Metadata account attached to it as a PDA. You can read more about this in the [Token Metadata program documentation](https://docs.metaplex.com/programs/token-metadata/overview).
+    
+## Buying asset using a custom SPL Token
+You can configure any SPL-token to act as the currency of your own Auction House (as opposed to defaulting to using native SOL).
+
+This can be achieved by setting the `treasuryMint` parameter in the Auction House Instance Account to the mint account of the SPL-token of your liking.
+
+## Custom Order Matching
+As mentioned before, Auction House allows marketplaces to implement their own custom order matching algorithms. This is done using the 3rd and the final Trade State in the Auction House protocol: the `FreeSellerTradeState`.
+
+If the buyer intentionally lists their NFT for a price of 0, a new `FreeSellerTradeState` is made. The Auction House can then change the sale price to match a matching Bid that is greater than 0. This allows the Auction house to do complicated order matching to find the best price for the seller.
+
+## Auctioneer
+The current Auction House implementation is designed with instant sales in mind and currently has no features that enable the various auction types that have become popular in the Solana ecosystem (ex. ***Timed auctions***) 
+
+Auctioneer is a customized contract type, written by the user, that uses the composability pattern of Auction House to control an individual Auction House instance. 
+
+To fully enable Auctioneer to use an Auction House instance's instructions, it must be explicitly delegated. 
+
+![](https://i.imgur.com/RyZUfR9.png)
+
+## Next steps
+In this page we have gone through the very basics of the Auction House protocol and the power it possesses. There is whole lot more Auction House is capable of and a lot more that can be configured to your liking.
+
+To get a good understanding of how the Auction House works, we first need to dive a little deep into the Accounts and the Instructions that are are a part of the Auction House. Kindly go through these pages to get a good unserstanding of the above:
+* [Getting Started]()
+* [Accounts]()
+* [Instructions]()
+* [Executing Sales]()
+* [Cancelling Bids and Listings]()
```

Comment:
I think we should introduce "Getting Started", "Accounts" and "Instructions" separately to the other two. The formers are repeated on every program documentation and act as human-friendly API references whereas the latters are feature pages specific to Auction House.